src/super_gradients/training/models/segmentation_models/segformer.py

```diff
@@ -347,6 +354,12 @@ def __init__(
 
         self.init_params()
 
+        self.num_classes = num_classes
+
+        self.use_sliding_window_validation = False
+        self.sliding_window_crop_size = sliding_window_crop_size
```

Comment:
Please convert them to tuples before saving in self, as they my have a type of ListConfig, which is not desirable to be a part of the model: `self.sliding_window_crop_size = tuple(sliding_window_crop_size)`

Reply:
Fixed on 4e91c8d.